### DISCUSSION TITLE
What is the "the suzanne model used by Blender"?

### DISCUSSION THREAD

**Discussion by Clarkszw** [fury-gl/fury#752] [Q&A] [answered]
I am going through the content on the website and currently on the page of [Get started](https://fury.gl/latest/getting_started.html)

In the beginning of the section, it says "Let us include the `suzanne` model used by Blender".

What is the `suzanne` model used by Blender? Should I create it myself to continue the tutorial?

Thanks in advance:)

**Comment by skoudoro** [comment, marked as answer]
Good catch!, you can create a PR to add the step to get `suzanne` model. See below

```python
from fury.data import fetch_viz_models

models = fetch_viz_models()
# in the models dictionnary, you have suzanne model path


```

**Reply by Clarkszw** [reply]
It is a surprise to me that this turns to be a test 😂. 

I was checking the Wikipedia and rushing back to answer myself to cover this basic knowledge:

[A low-polygon model with only 500 faces, Suzanne is included in Blender and often used as a quick and easy way to test materials, animations, rigs, textures, and lighting setups. It is as easily added to a scene as primitives such as a cube or plane. The largest Blender contest gives out an award called the Suzanne Award.](https://en.wikipedia.org/wiki/Blender_(software)#:~:text=A%20low-polygon%20model%20with%20only%20500%20faces%2C%20Suzanne,gives%20out%20an%20award%20called%20the%20Suzanne%20Award.)

Of course, I will try to create a PR to add this step. will comeback if I get stuck;)

**Reply by Clarkszw** [reply]
Where can I find source code for this [Getting_started page](https://fury.gl/latest/_sources/getting_started.rst.txt) in the repo?
I am thinking modify the code and create a PR to add this step in the page. But I can't find where is located in the repo..
